fitnessllm_dataplatform/utils/reques_utils.py

```
@@ -0,0 +1,17 @@
+import requests
+
+
+def handle_status_code(response: requests.models.Response) -> dict:
+    """Handle status code."""
+    if response.status_code == 200:
+        return response.json()
+    elif response.status_code == 400:
```

Ruff (0.8.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)